concourse/pipelines/templates/jobs/promote-pxf-artifacts-tpl.yml

```
@@ -0,0 +1,38 @@
+{% set name = 'promote-pxf-artifacts' %}
+{% do x.update({'job_name': name}) %}
+- name: [[name]]
+  plan:
+  - in_parallel:
+    - get: pxf_src
+      resource: pxf-src
+      trigger: true
+      passed: [[job_names.tests['backward_compatibility']]]
+{% for tarball in job_names.tarballs.keys() | list %}
+    - get: [[tarball]]
+      passed: [[job_names.tarballs[tarball]]]
+{% endfor %}
+    - get: google-cloud-sdk-slim-image
+  - task: [[name]]
+    image: google-cloud-sdk-slim-image
+    params:
+      ENVIRONMENT: [[ environment ]]
```

```suggestion
      ENVIRONMENT: [[environment]]
```

Any reason to add spaces around the variable name?

---

Nope. Just carried on from the original pipeline.